Initium-Odp/war/odp/javascript/script.js

```diff
@@ -1481,23 +1483,28 @@ $(document).keyup(function(event){
 		var buttonToPress = $('[shortcut="'+event.which+'"]');
 		if (buttonToPress.length>0)
 		{
-			window.disableShortcuts = true;
+			if (buttonToPress[0].hasClass("main-button-half")==false)
+				window.disableShortcuts = true;
```

Comment:
It keeps the shortcuts available if a key for one of the Nearby XXX popups was used. Previously, opening one of these popups and then closing it left you stuck without shortcuts. I also thought this was a nicer way to do it than reenabling on close, as this way you can still use the shortcuts while the popup is open (Ex: Done reordering your house inventory? Just press 1 and you're back in Aera!)